components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -168,6 +172,23 @@ open class GleanInternalAPI {
         }
     }
 
+    /**
+     * Records a list of the currently enabled accessibility services.
+     *
+     * https://developer.android.com/reference/android/view/accessibility/AccessibilityManager.html
+     * @param accessibilityManager The system's AccessibilityManager as returned from applicationContext.getSystemService
```

Comment:
nit: I think Android's api objects can be referenced in the docs by using `[AccessibilityManager]` (adding the square brackets)